### PR TITLE
Changes to lower-tier disease transmission

### PR DESCRIPTION
**Pull request by zeskorion**
## About The Pull Request
-Adds a new transmission type: bloodborne. This type should only infect if the disease is injected or otherwise called with force_contract_disease
-infective vectors need Blood or higher transmission to spread
- buff to sentient disease's secrete infection to compensate

## Why It's Good For The Game

makes low-transmission diseases a bit easier to control

## Changelog
:cl:
add: Bloodborne transmission
tweak: infectivity of items with the infective component
balance: sentient disease secrete infection is now robust, but only if the disease has sufficient transmission
/:cl:

### CODE REVIEW

File: code/__DEFINES/diseases.dm [hunk 1]
```diff
@@ -12,8 +12,9 @@
 #define CAN_RESIST	(1<<2)
 
 // Spread Flags
-#define DISEASE_SPREAD_SPECIAL			(1<<0)
-#define DISEASE_SPREAD_NON_CONTAGIOUS	(1<<1)
+#define DISEASE_SPREAD_SPECIAL			(1<<-1)
```
Comment (qwertyquerty) [inline code/__DEFINES/diseases.dm:15]: `(1<<-1)`

You uhhh, can't do this. All shifts must be positive.

Comment (qwertyquerty) [reply code/__DEFINES/diseases.dm:15]: Hello sir?

Comment (zeskorion) [reply code/__DEFINES/diseases.dm:15]: sorry, aint been on in awhile. I'll get to it in a bit.

**Comment by zeskorion** [comment]
I was just looking over this and realized some changes i made were never comitted, this just adds a new transmission type that does nothing, gonna close it for the time being.

**Pull request closed without merging**